mbuild/tests/test_compound.py

```diff
@@ -1082,3 +1084,16 @@ def test_get_smiles(self):
         for test_string in test_strings:
             my_cmp = mb.load(test_string, smiles=True)
             assert my_cmp.get_smiles() == test_string
+
+    def test_sdf(self, methane):
+        methane.save('methane.sdf')
+        sdf_string = mb.load('methane.sdf')
+
+        assert np.allclose(methane.xyz, sdf_string.xyz, atol=1e-5)
+
+    def test_multiple_sdf(self, methane):
+        filled = mb.fill_box(methane, n_compounds=10, box=[0, 0, 0, 4, 4, 4])
+        filled.save('methane.sdf')
+        sdf_string = mb.load('methane.sdf')
+
+        assert np.allclose(filled.xyz, sdf_string.xyz, atol=1e-5)
```

Comment:
This also confuses me: if `methane.sdf` represents 10 methane molecules, wouldn't it be read back in only with only one methane molecule? In which case we'd compare `assert np.allclose(methane.xyz, sdf_string.xyz, atol=1e-5)`

Reply:
I _think_ the whole box is considered the `pybel` molecule.